mainnet/singularity/Dockerfile1

```diff
@@ -0,0 +1,14 @@
+FROM quarkchaindocker/pyquarkchain:mainnet1.6.1
```

Comment:
What is this extra Docfile for?

Reply:
one is build from begining, this one is build from last docker.